platforms/documentation/docs/src/docs/design/gradle-module-metadata-latest-specification.md

```diff
@@ -8,17 +8,17 @@ Gradle publishes Gradle Module Metadata along with traditional metadata. Gradle
 
 This document describes version 1.1 of the Gradle module metadata file format. A module metadata file describes the contents of a _module_, which is the unit of publication for a particular repository format, such as a module in a Maven repository. This is often called a "package" in many repository formats.
 
-The module metadata file is a JSON file published alongside the existing repository specific metadata files, such as a Maven POM or Ivy descriptor. It adds additional metadata that can be used by Gradle versions and other tooling that understand the format. This allows the rich Gradle model to be mapped to and "tunnelled" through existing repository formats, while continuing to support existing Gradle versions and tooling that does not understand the format. 
+The module metadata file is a JSON file published alongside the existing repository-specific metadata files, such as a Maven POM or Ivy descriptor. It adds additional metadata that can be used by Gradle versions and other tooling that understand the format. This allows the rich Gradle model to be mapped to and "tunneled" through existing repository formats while continuing to support existing Gradle versions and tooling that do not understand the format. 
 
-The module metadata file is intended to be machine generated rather than written by a human, but is intended to be human readable.
+The module metadata file is intended to be machine-generated rather than written by a human but is intended to be humanly readable.
```

Comment:
💅 
```suggestion
The module metadata file is intended to be machine-generated rather than written by a human but is intended to be human-readable.
```